ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -879,6 +879,8 @@ private void writeMessagesToLocalStoreAndAdvanceTokens(List<ExchangeMetadataResp
               totalBlobsFixed += messageInfoList.size();
               remoteReplicaInfo.setToken(exchangeMetadataResponse.remoteToken);
               remoteReplicaInfo.setLocalLagFromRemoteInBytes(exchangeMetadataResponse.localLagFromRemoteInBytes);
+              replicationMetrics.updateLagMetricForRemoteReplica(remoteReplicaInfo,
```

Comment:
Can we update the replication test to verify that the new metrics are getting populated?

Reply:
Test added.